docs/content/en/tasks/backup-restore/_index.md

```diff
@@ -154,58 +149,199 @@ status:
 ```
 
 All pods having completed the backup will be in the `finished` list.
+At the end of the backup operation, a `MedusaBackup` custom resource will be created with the same name as the `MedusaBackupJob` object. It materializes the backup locally on the Kubernetes cluster.
+For a restore to be possible, a `MedusaBackup` object should exist.
+
+
+## Creating a Backup Schedule
+
+K8ssandra-operator v1.2 introduced a new `MedusaBackupSchedule` CRD to manage backup schedules using a [cron expression](https://crontab.guru/):  
+
+```yaml
+apiVersion: medusa.k8ssandra.io/v1alpha1
+kind: MedusaBackupSchedule
+metadata:
+  name: medusa-backup-schedule
+  namespace: k8ssandra-operator
+spec:
+  backupSpec:
+    backupType: differential
+    cassandraDatacenter: dc1
+  cronSchedule: 30 1 * * *
+  disabled: false
+```
+
+The above definition would trigger a differential backup of `dc1` every day at 1:30 AM. The status of the backup schedule will be updated with the last execution and next execution times:
```

Comment:
Somewhere in the section I would be explicit about where the CR needs to be created, i.e., same namespace and cluster where the CassandraDatacenter exists.

Reply:
👍